Packs/SymantecEndpointProtection/Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py

```diff
@@ -801,19 +898,407 @@ def move_client_to_group_command(token):
         })
 
 
+def fetch_incidents(token):
+    json_response = do_get(token, False, 'sepm/api/v1/events/critical')
```

Comment:
This can be problematic as the command doesn't take any filtering arguments. we don't want the fetch to handle too much data at the same time.
 please check if there is a way to use paging in this command (using page_size and page_number arg) or if the amount of data received is manageable.